src/shared/components/common/password-input.tsx

```diff
@@ -99,8 +99,10 @@ class PasswordInput extends Component<PasswordInputProps, PasswordInputState> {
                 onInput={onInput}
                 value={value}
                 required
-                pattern=".{10,60}"
+                pattern=".+"
```

Comment:
I changed the pattern since length is all we're checking for and min and max length gives more information to users as to what is wrong than using the pattern.